reporter/src/main/kotlin/reporters/evaluatedmodel/EvaluatedModelMapper.kt

```diff
@@ -445,13 +455,27 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
                 issues += addIssues(this.issues, EvaluatedOrtIssueType.ANALYZER, dependency, null, packagePath)
             }
 
+            val children = visitDependencies { dependencies ->
+                dependencies.map { node ->
+                    if (!deduplicateDependencyTree || node.getInternalId() !in visitedNodes) {
+                        val childNode = node.toEvaluatedTreeNode(scope, path + dependency)
+                        visitedNodes += node.getInternalId() to childNode
```

Comment:
Do I understand it correctly that subtrees in the graph are already deduplicated (as the logic uses `node.getInternalId()` for de-duplication), so that it only prevents duplicating subtrees from the graph but does not de-duplicate subtrees from the graph?

Reply:
Yes, in the graph all subtrees have already been deduplicated.

Reply:
optional: maybe this would be an interresting note to add to the commit message?

Reply:
The commit message has been extended accordingly.

Reply:
Thanks!